__tests__/components/ui/SideNavigation/_partials/Action.test.tsx

```diff
@@ -0,0 +1,67 @@
+import { ActivableLink } from "@/components/ui/SideNavigation/_partials/Action";
+import "@testing-library/jest-dom";
+import { fireEvent, render, screen } from "@testing-library/react";
+
+describe("Activable link", () => {
+  const item = {
+    subitems: [],
+    link: "/example",
+  };
+  it("should render the link with subitems", () => {
+    render(
+      <ActivableLink
+        item={{
+          subitems: [{ label: "subitem", link: "subitem link" }],
+          link: "/dummy",
+        }}
+      >
+        Link test
+      </ActivableLink>
+    );
+    const link = screen.getByTestId("activableLinkId");
+    expect(link).toBeInTheDocument();
+    expect(link.textContent).toBe("Link test");
+  });
+
+  it("should render the link with no subitems", () => {
+    render(<ActivableLink item={item}>Link test</ActivableLink>);
+    const link = screen.getByText("Link test");
+    expect(link.tagName).toBe("A");
+    expect(link).toHaveAttribute("href", item.link);
+  });
+
+  it("should render the link with activable link style", () => {
+    render(
+      <ActivableLink
+        item={{
+          subitems: [{ label: "subitem", link: "subitem link" }],
+          link: "/dummy",
+        }}
+        isActive
+      >
+        Link test
+      </ActivableLink>
+    );
+    const link = screen.getByTestId("activableLinkId");
+    expect(link.children[0].tagName).toBe("SPAN");
+    expect(link.children[0].className).toContain("activable-link");
```

Comment:
let's avoid testing classes, the name of this call can change while the component is still behaving the same way.

Reply:
Still have this